ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift

```diff
@@ -26,13 +26,15 @@ import UIKit
     }
 
     @objc
-    public func presentPaywall() {
+    public func presentPaywall(
+        displayCloseButton: Bool = false
```

Comment:
I wonder if we should make it nullable instead? So we can use the default defined in `purchases-ios`

Reply:
Actually, not sure if that would work in objc, since nullable booleans didn't work well if I remember correctly

Reply:
I tried and I get 

`Method cannot be marked @objc because the type of the parameter cannot be represented in Objective-C`

I can maybe do two functions?